Makefile

```
@@ -54,7 +54,7 @@ clean-docs:
 
 install:
 	go install ${LDFLAGS}
-	mv ${GOPATH}/bin/packet-cli ${GOPATH}/bin/packet
+	mv ${GOPATH}/bin/${BINARY} ${GOPATH}/bin/${BINARY}
```

```
$ make install
go install -ldflags "-X github.com/equinix/metal-cli/cmd.Version=f698724 -X github.com/equinix/metal-cli/cmd.Build=`git rev-parse --short HEAD`"
mv /Users/marques/.local/share/go/bin/metal /Users/marques/.local/share/go/bin/metal
mv: rename /Users/marques/.local/share/go/bin/metal to /Users/marques/.local/share/go/bin/metal: No such file or directory
make: *** [install] Error 1
```


```suggestion
	mv ${GOPATH}/bin/metal-cli ${GOPATH}/bin/${BINARY}
```

Perhaps we can create a new variable (`PROJECT_NAME`?) that is used by `PACKAGE_NAME` and this `mv` command.